### ISSUE TITLE
Broken (missing) link in README

### ISSUE THREAD

**Issue by HorridModz** [aaronedev/aaronedev#4]
[Line 29](https://github.com/aaronedev/aaronedev/blob/d9b75b02439bcbac626b198f61db0442305cdc8d/README.md?plain=1#L29) of the README is a link to `PowerShell Profile Configuration`. However, the link is simply `#`:

```md
[PowerShell Profile Configuration](#)
```

This causes the link to do nothing, and I don't see how it could be interventional, especially when the other link (`Violet Void Theme`) points to an actual website (`https://violet-void-theme.com`). I'm guessing that you meant to put a website or github repository in the former link, but accidentally left it blank.

And by the way, as I mentioned in another issue, the `https://violet-void-theme.com` is also broken - the website gives a 403 Forbidden error.

**Comment by aaronedev** [comment]
@HorridModz thank you for pointing that out! You're correct—the link is currently just a placeholder. I’m still working on the first release of the PowerShell Profile configuration, which will be included as part of the Violet Void theme. I've created a repository for the theme, and I plan to create a separate one specifically for the PowerShell Profile configuration soon.

It's not quite ready for public release yet, but I’ll be posting updates as I make progress. I appreciate you bringing this to my attention, and I’ll ensure the link is updated once everything is in place.

Furthermore, checkout my reply for issue #2.

**Comment by HorridModz** [comment]
> @HorridModz thank you for pointing that out! You're correct—the link is currently just a placeholder. I’m still working on the first release of the PowerShell Profile configuration, which will be included as part of the Violet Void theme. I've created a repository for the theme, and I plan to create a separate one specifically for the PowerShell Profile configuration soon.
> 
> It's not quite ready for public release yet, but I’ll be posting updates as I make progress. I appreciate you bringing this to my attention, and I’ll ensure the link is updated once everything is in place.
> 
> Furthermore, checkout my reply for issue #2.

Hmm, you gonna fix the links? No nudging intended, but might as well make them do something! Again, it's your choice though.

**Comment by aaronedev** [comment]
I fixed it, thank you for pointing out. I was quite busy working on something but now taking a break 🌶️
